ecl/eclccserver/eclccserver.cpp

```diff
@@ -429,7 +429,10 @@ class EclccCompileThread : implements IPooledThread, implements IErrorReporter,
                 if (!workunit->getDebugValueBool("obfuscateOutput", false))
                 {
                     Owned<IWUQuery> query = workunit->updateQuery();
-                    query->setQueryText(eclQuery.s.str());
+                    if (getArchiveXMLFromFile(realdllfilename, wuXML.clear()))  // MORE - if what was submitted was an archive, this is probably pointless?
```

Comment:
Would it be better to go and get the archive from the workunit in esp, rather than filling dali with the archive?

Reply:
Yes, probably, though that's arguably a separate issue. I can look at doing so